src/Stoa.ts

```diff
@@ -381,59 +381,109 @@ class Stoa extends WebService
     }
 
     /**
-     * GET /wallet/transactions/history/:addresses
-     *
-     * Called when a request is received through the `/wallet/transactions/history/:addresses` handler
-     * The parameter `addresses` are the form of multiple addresses separated by `,`.
-     * The parameter `pageSize` is the maximum size that can be obtained from one query, default is 10
-     * The parameter `page` is the number on the page, this value begins with 1, default is 1
+     * GET /wallet/transactions/history/:address
      *
+     * Called when a request is received through the `/wallet/transactions/history/:address` handler
+     * ```
+     * The parameter `address` are the address to query.
+     * The parameter `pageSize` is the maximum size that can be obtained
+     *      from one query, default is 10
+     * The parameter `page` is the number on the page, this value begins with 1,
+     *      default is 1
+     * The parameter `type` is the type of transaction to query.
+     *      This can include multiple types.
+     *      Transaction types include "inbound", "outbound", "freeze", "payload".
+     *      The "inbound" is an increased transaction of funds at the address.
+     *      The "outbound" is a transaction with reduced funds at the address.
+     *      Users can select only "inbound", "outbound".
+     *      The "freeze", "payload" are always included.
+     *      default is "inbound,outbound,freeze,payload"
+     * The parameter `beginDate` is the start date of the range of dates to look up.
+     * The parameter `endDate` is the end date of the range of dates to look up.
+     * The parameter `peer` is used when users want to look up only specific
+     *      address of their counterparts.
+     *      Peer is the withdrawal address in the inbound transaction and
+     *      a deposit address in the outbound transaction
      * Returns a set of transactions history of the addresses.
+     * ```
      */
     private getWalletTransactionsHistory (req: express.Request, res: express.Response)
     {
-        let params_addresses: string = String(req.params.addresses);
+        let address: string = String(req.params.address);
 
-        logger.http(`GET /wallet/transactions/history/${params_addresses}}`);
+        logger.http(`GET /wallet/transactions/history/${address}}`);
 
-        let page_size = (req.query.pageSize !== undefined)
-            ? Number(req.query.pageSize.toString())
-            : 10;
+        if ((req.query.beginDate !== undefined) &&
+            !Utils.isPositiveInteger(req.query.beginDate.toString()))
```

Comment:
Can it verify the correct date integer format? Validates only integer formats.
An error must occur if the start date is greater than the end date. or the opposite case
What if the Genesis Block is less than the start time?

Reply:
>> What if the Genesis Block is less than the start time?
I don't think this will be a big problem.

Reply:
Yes, You can handle it later, but you need to check the validity of the date.

Reply:
>> An error must occur if the start date is greater than the end date

That's a good suggestion. I'll apply it.

Reply:
But you know this is different from other types, right?

Reply:
But it's clear that suggestions are one of the best ways.